cmd/root.go

```diff
@@ -39,6 +40,7 @@ func ServerFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&publicEndpoint, "public-endpoint", "http://localhost:8080", "Public endpoint that this instance is exposed under")
 	flags.StringVar(&apmHub, "apm-hub", "http://apm-hub:8080", "APM Hub URL")
 	flags.StringVar(&configDb, "config-db", "http://config-db:8080", "Config DB URL")
+	flags.StringVar(&snapshot.CanaryCheckerURL, "canary-checker", "http://canary-checker:8080", "Canary Checker URL")
```

Comment:
Why can't we just use api.CanaryChckerPath ?

Reply:
By bad, I didn't see that flag at first, will be using that now